src/Components/RelevanceInspector.tsx

```diff
@@ -141,23 +128,17 @@ export default class RelevanceInspector extends React.Component {
             {this.getExecuteQueryControl()}
           </FormGroup>
         </Grid>
-        {this.headlessRelevanceInspector.state.isEnabled ? (
-          <>
+        {this.headlessRelevanceInspector.state.isEnabled && (
+          <Grid component="label" container alignItems="center" spacing={1}>
+            <Grid item>Print debug info: </Grid>
             <Avatar style={this.avatarStyle}>
               <BugReportIcon
-                onClick={(e) => {
-                  this.setDebugWindow();
+                onClick={() => {
+                  console.log(this.headlessRelevanceInspector.state);
```

Comment:
I also tried using [the dedicated function](https://docs.coveo.com/en/headless/latest/reference/controllers/relevance-inspector/#loginformation) but it didn't quite seem to work for some reason 🤔 
Any idea?